webapp/channels/src/utils/constants.tsx

```diff
@@ -2024,7 +2027,7 @@ export const Constants = {
     MIN_TEAMNAME_LENGTH: 2,
     MAX_TEAMNAME_LENGTH: 64,
     MAX_TEAMDESCRIPTION_LENGTH: 50,
-    MIN_CHANNELNAME_LENGTH: 1,
+    MIN_CHANNELNAME_LENGTH: 2,
```

Comment:
Why change this? I don't know why someone would need a 1 character channel name (maybe a language like Chinese?), but the previous value matches a constant on the server that would need to be changed if we're changing this as well

Reply:
there were a lot of inconsistencies across the webapp with the min channel name lenght value. @matthewbirtch should we keep the min channel lenght as 1 or as 2 as shown in the design?

Reply:
I don't see a reason to make the change. Let's leave it as it was. I think we had it in the figma file as an example error message, but we didn't define a min length at all in the [UX docuementation](https://mattermost.atlassian.net/wiki/spaces/DES/pages/2260631553/UX+Spec+Channel+Settings+Modal). I'll update Figma to reflect this.

Reply:
Reminder to revert this one